modules/core/src/main/java/org/apache/ignite/internal/pagemem/store/PageStoreCollection.java

```diff
@@ -33,8 +33,7 @@ public interface PageStoreCollection {
 
     /**
      * @param grpId Cache group ID.
-     * @return Collection of related page stores.
-     * @throws IgniteCheckedException If failed.
+     * @return Collection of related page stores or {@code null} if it's not found.
```

Comment:
Does it make sense to return an empty collection instead of `null`?

Reply:
I wouldn't do that. Empty collection means "there's something and it's empty", while null means "there's nothing". Also, we already use nulls there